src/Persistence.Linnapps/Persistence.Linnapps.csproj

```diff
@@ -6,8 +6,18 @@
     <RootNamespace>Linn.Products.Persistence.Linnapps</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Linn.Common.Configuration" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.2.0" />
```

Comment:
where is logging.debug being used is it part of a different package that requires it